app/src/main/java/com/example/universitieslist/view/components/UniversityList.kt

```diff
@@ -18,11 +21,12 @@ fun UniversityList() {
                 bottom = dimens20,
             )
     ) {
-        items(count = 15) {
+        items(count = universityList.size) { index ->
+            val university = universityList[index]
             UniversityItem(
-                schoolName = "African University of Science and Technology Coming to an end",
-                website = "http://www.aust-abuja.org/",
-                country = "Nigeria"
+                schoolName = university.name.orEmpty(),
```

Comment:
I'm curious to know how this displays when all fields are empty. Will that be an empty card?

Reply:
I've handled it using a let `universityList?.let {}` so if all fields are empty, the card will not show.